### ISSUE TITLE
WebUI stopped working for me

### ISSUE THREAD

**Issue by namenamae** [Engineer-of-Stuff/stable-diffusion-paperspace#1]
![what's going on](https://user-images.githubusercontent.com/3712767/192775475-fd963ece-643e-45d8-8988-c1e9bfee6f09.png)
No matter what I do, I get this.

**Comment by Engineer-of-Stuff** [comment]
That's a pickle error. Pickle is used to create the model checkpoints. Try deleting and redownloading the .ckpt file

**Comment by namenamae** [comment]
I've tried this, but I keep getting the same error.

**Comment by namenamae** [comment]
Here's a screenshot what I did on my latest attempt. I have lost track of how many times I have deleted the whole project to try and start again.

![everything I did](https://user-images.githubusercontent.com/3712767/192920251-b3dab61f-e09d-413e-af2a-85ec5e14b01e.png)

**Comment by Cyberes** [comment]
You might have run out of storage space. Try deleting all your files in `/notebooks/` and `/storage`. I added a block in the Tools section to reset your storage space or you can delete your notebook and start over.

Honestly, I'm not sure what can be done about the 5GB storage limit on the free tier. There are some potential options in the Filesize and Storage Disclaimer under Download the Model.

Can you open the terminal (the terminal icon in the left vertical menu) and run this command?

`df -h`

This is what mine gives. I'm on a paid plan.

```bash
Filesystem                                                                                                                                                                      Size  Used Avail Use% Mounted on
overlay                                                                                                                                                                         246G  183G   53G  78% /
tmpfs                                                                                                                                                                            64M     0   64M   0% /dev
tmpfs                                                                                                                                                                            15G     0   15G   0% /sys/fs/cgroup
10.0.3.159:6789,10.0.3.160:6789,10.0.3.161:6789,10.0.3.153:6789,10.0.3.158:6789:/                                                                                               187T  150T   38T  81% /notebooks
10.0.3.159:6789,10.0.3.160:6789,10.0.3.161:6789,10.0.3.153:6789,10.0.3.158:6789:/volumes/csi/csi-vol-71b3e44f-6502-11ec-88fd-deb75cecf6b1/a2f27a64-2679-4924-a0af-4aaf9cd77af1  500G  9.9G  491G   2% /storage
/dev/mapper/ubuntu--vg-root                                                                                                                                                     246G  183G   53G  78% /datasets
tmpfs                                                                                                                                                                            15G   22M   15G   1% /dev/shm
tmpfs                                                                                                                                                                            15G   12K   15G   1% /run/secrets/kubernetes.io/serviceaccount
tmpfs                                                                                                                                                                            15G   12K   15G   1% /proc/driver/nvidia
tmpfs                                                                                                                                                                           3.0G   29M  3.0G   1% /run/nvidia-persistenced/socket
udev                                                                                                                                                                             15G     0   15G   0% /dev/nvidia0
tmpfs                                                                                                                                                                            15G     0   15G   0% /proc/acpi
tmpfs                                                                                                                                                                            15G     0   15G   0% /proc/scsi
tmpfs                                                                                                                                                                            15G     0   15G   0% /sys/firmware
```
You're interested in the `/notebook` and `/storage` lines. Maybe we can try downloading the models to a weird directory that doesn't count towards the storage limit?

**Comment by Cyberes** [comment]
Fixed in 36d5e96

**Comment by Cyberes** [comment]
# If anyone is coming here from Google, [please read this!!!!!!!!!!!!!!!11](https://github.com/Engineer-of-Stuff/stable-diffusion-paperspace/issues/2#issuecomment-1271044373)
